.lintrunner.toml

```diff
@@ -47,7 +47,7 @@ init_command = [
     'mccabe==0.7.0',
     'pycodestyle==2.11.1',
     'pyflakes==3.1.0',
-    'torchfix==0.4.0 ; python_version >= "3.9"',
```

Comment:
This should remain pinned so we can explicitly update it

Reply:
OK